environment.yml

```diff
@@ -1,25 +1,505 @@
+name: sophie2
 channels:
-   - conda-forge
-   - bioconda
-   - defaults
+  - r
+  - HCC
+  - conda-forge
+  - bioconda
+  - defaults
 dependencies:
```

Comment:
Is it normal to have this many dependencies? It's way more than I've ever seen, even for a R conda file.

Reply:
no definitely not. What people usually do is only record packages that they're going to install. I got nervous in this case though that the environment is really specific and a lot of these packages move very quickly, so I just recorded everything -- all of the packages I installed plus all of their dependencies. This will make it more brittle across operating systems, but I thought that was better than under-documenting and not being able to reproduce the compute state